molecule/default/requirements.yml

```diff
@@ -1,9 +1,5 @@
 ---
 - src: https://github.com/cisagov/ansible-role-cyhy-core
   name: cyhy_core
-- src: https://github.com/cisagov/ansible-role-pip
-  name: pip
-- src: https://github.com/cisagov/ansible-role-python
-  name: python
```

Comment:
Since this role installs a package with `pip2` should we add these as role dependencies (with appropriate variables for Python 2 support)?

Reply:
Well they do get pulled in already via the `cyhy_core` dependency.  But yes, we should probably add them.  Doing it now.

Reply:
Resolved in [336680e](https://github.com/cisagov/ansible-role-ncats-webd/pull/63/commits/336680e96e66f26d6a2f5f74dbfb9fb9ffc6b8a3).

Reply:
Please see commit 336680e.